backend/tracim_backend/lib/core/content.py

```diff
@@ -1583,7 +1586,7 @@ def get_preview_page_nb(self, revision_id: int, file_extension: str) -> typing.O
                 file_path,
                 file_ext=file_extension
             )
-        except UnsupportedMimeType:
+        except Exception:
```

Comment:
I suggest to continue to except `UnsupportedMimeType` (and return None in that case) and except Exception (and log a warning about the error).

Reply:
done